tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -49,29 +85,93 @@ describe('Header', () => {
     expect(menuBtn).toExist()
   })
 
-  it("should add usa-current class to Welcome tab when on '/edit-profile'", () => {
-    const store = mockStore(initialState)
+  it("should add usa-current class to Welcome tab when on '/welcome'", () => {
```

Comment:
Adding a test for the new path of `/welcome` so that the `Welcome` navigation tab and the `Profile` navigation tab don't show active at the same time.